client/src/components/AppContainer/AppContainer.js

```diff
@@ -1,19 +1,21 @@
 import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import AuthRoute from "./AuthRoute";
-import PrivateRoute from "./PrivateRoute";
-import AuthNavbar from "./AuthNavbar";
-import UnAuthNavbar from "./UnAuthNavbar";
-import Home from "./Home";
-import Signup from "./Signup";
-import Signin from "./Signin";
-import User from "./User";
+import AuthRoute from "../AuthRoute/AuthRoute.js";
+import PrivateRoute from "../PrivateRoute/PrivateRoute.js";
+import AuthNavbar from "../AuthNavbar/AuthNavbar";
+import UnAuthNavbar from "../UnAuthNavbar/UnAuthNavbar.js";
+import Home from "../Home/Home.js";
+import Signup from "../Signup/Signup.js";
+import Signin from "../Signin/Signin.js";
+import User from "../User/User.js";
+// import style from './AppContainer.css'
```

Comment:
If we are just going to not use the commented out code anymore than we should just delete it all together.